### PR TITLE
docs: add example Docker Compose files

### PR DESCRIPTION
**Pull request by justo-mend**

### CODE REVIEW

**Review by rarkins — CHANGES_REQUESTED** [review]
This needs an accompanying documentation to describe it

File: examples/renovate-ee-simple.yml
Comment (rarkins) [inline examples/renovate-ee-simple.yml:1]: These files are under an "examples" folder but it seems like they'd benefit from an extra folder layer, such as "docker-compose" here.

Comment (justo-mend) [reply examples/renovate-ee-simple.yml:1]: Good thinking. Pushed in new commit.

File: examples/env/github.env
Comment (rarkins) [inline examples/env/github.env:1]: Where are these two env files referenced?

**Review by rarkins — CHANGES_REQUESTED** [review]
This still feels too much like a "file dump" and not a cohesive example. Ideally has:
- Markdown docs describing what each example is for
- References between files, e.g. to the .env files

File: examples/env/github.env
Comment (justo-mend) [reply examples/env/github.env:1]: There were referenced in earlier versions of the ee yml. But they are no longer directly linked. I could remove them. Or, add example config for the other platforms alongside the github one.

**Comment by justo-mend** [comment]
> This still feels too much like a "file dump" and not a cohesive example. Ideally has:
> 
> * Markdown docs describing what each example is for
> * References between files, e.g. to the .env files

Good idea. I'll create a readme at the 'examples' level.

File: examples/env/github.env
Comment (rarkins) [reply examples/env/github.env:1]: If we have undocumented examples then it risks being more of a confusion than help. Would be best to have them tied together with some text

Comment (justo-mend) [reply examples/env/github.env:1]: I've added some text to the new Examples readme. Maybe that helps.

**Review by rarkins — APPROVED** [review]

**Pull request merged**